### PR TITLE
docs(README): add demo reference

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,9 @@ As with Bootstrap 4 and Angular 2, this library is a work in progress. Please ch
...
 
-We support the same browsers and versions supported by both Bootstrap 4 and Angular 2, whichever is _more_ restrictive.
-See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-date Angular 2 browser support.
-
-* Chrome (45+)
-* Firefox (40+)
-* IE (9+) (but see [Bootstrap 4's notes](http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#internet-explorer-9) on IE9)
-* Edge (20+)
-* Safari (7+)
+Check all the directives in action at https://ng-bootstrap.github.io
```
Comment (Foxandxss) [inline README.md]: I don't mind, but the main place to put the URL is on the top of the repo. I will do that.

Comment (Foxandxss) [inline README.md]: Oh, that is done.

File: README.md [hunk 2]
```diff
@@ -73,6 +66,17 @@ export class App {
...
 ```
+## Supported browsers
+
+We support the same browsers and versions supported by both Bootstrap 4 and Angular 2, whichever is _more_ restrictive.
+See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-date Angular 2 browser support.
+
+* Chrome (45+)
+* Firefox (40+)
+* IE (9+) (but see [Bootstrap 4's notes](http://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#internet-explorer-9) on IE9)
+* Edge (20+)
```
Comment (wesleycho) [inline README.md]: 20+? Isn't Edge on 14 or 15 or something like that?

**Comment by wesleycho** [comment]
This LGTM - I had a comment about Edge versions, but I didn't realize it updated versions so fast.

**Pull request closed without merging**